src/status_im2/contexts/wallet/common/screen_base/create_or_edit_account/view.cljs

```diff
@@ -45,12 +46,21 @@
          :i/reaction]]
        [quo/title-input
         {:placeholder     (i18n/label :t/account-name-input-placeholder)
-         :max-length      24
+         :max-length      20
```

Comment:
let's put this in a constant

Reply:
Yes, we should. I forgot to move it to constant. Thanks :)